plugins/build.gradle

```diff
@@ -176,3 +214,34 @@ task publishIndex( type: io.nextflow.gradle.tasks.GithubRepositoryPublisher ) {
     githubEmail = github_commit_email
     githubToken = github_access_token
 }
+
+/*
+ * Task to install all plugins to the Nextflow plugins directory
+ */
+task installPlugins() {
```

Comment:
Gradle isn't the most intuitive, but in this case these two tasks (`installPlugins` and `installSpecificPlugin`) are actually implied by `installPlugin`.

The command
```
./gradlew installPlugin
```
will execute the `installPlugin` task on all projects which define it, and since we define the `installPlugin` in the `submodules` block, it's automatically defined on all the individual plugin modules.

To execute the task only for a specific plugin, we can explicitly state which submodule to run the task on:
```
./gradlew :plugins:nf-hello:installPlugin
```

This means we can remove both of these tasks, and update the Makefile to use the above forms.

Reply:
Ah! I didn't know that! Thanks Tom!!